core/db/migrate/20200212144523_add_hide_from_nav_to_taxons.rb

```diff
@@ -0,0 +1,5 @@
+class AddHideFromNavToTaxons < ActiveRecord::Migration[6.0]
+  def change
+    add_column :spree_taxons, :hide_from_nav, :boolean, default: false
```

Comment:
imo, we also might want to add an index to this attribute. @pmilewski let us know what You think.